### PR TITLE
[NEW] event_sale_confirmation: hide button confirm in some cases

### PR DESCRIPTION
**Pull request by yelizariev**
Module hides ``[Confirm]`` button when ticket is not free and user doesn't have access level ``Events: Manager``.

### CODE REVIEW

File: event_sale_confirmation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "Restricted access to confirm unpaid tickets",
+    "summary": "Only event manager can confirm unpaid tickets",
+    "version": "10.0.1.0.0",
+    "category": "Event Management",
+    "website": "https://github.com/OCA/event",
+    "author": "IT-Projects LLC, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
```
Comment (pedrobaeza) [inline event_sale_confirmation/__manifest__.py]: Don't put empty keys

File: event_sale_confirmation/models/event_registration.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# Copyright 2017,2018 IT-Projects LLC - Ivan Yelizariev
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import models, fields, api
+
+
+class EventRegistration(models.Model):
+
+    _inherit = 'event.registration'
+
+    event_ticket_price = fields.Float(related='event_ticket_id.price')
```
Comment (pedrobaeza) [inline event_sale_confirmation/models/event_registration.py]: readonly=True for avoiding rewritings each time.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: event_sale_confirmation/models/event_registration.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class EventRegistration(models.Model):
+
+    _inherit = 'event.registration'
+
+    event_ticket_price = fields.Float(
+        related='event_ticket_id.price',
+        readonly=True)
+    user_is_manager = fields.Boolean(compute='_compute_user_is_manager')
```
Comment (chienandalu) [inline event_sale_confirmation/models/event_registration.py:14]: I'm not sure if something like `can_confirm` would be a more appropiate name for this field

**Review by chienandalu — APPROVED** [review]
Looks good to me :+1:

**Review by chienandalu — COMMENTED** [review]
It shouldn't be hard to do some tests for coverage

File: event_sale_confirmation/models/event_registration.py (hunk 3 above)
Comment (yelizariev) [reply event_sale_confirmation/models/event_registration.py:14]: It's not so. The user can confirm when he is a manager OR ticket is free

**Comment by yelizariev** [comment]
What shall I do with this warning;
```

event_sale_confirmation/static/src/js/tour.js:62: [W7903(javascript-lint), ] Unexpected console statement. [Error/no-console]
event_sale_confirmation/static/src/js/tour.js:77: [W7903(javascript-lint), ] Unexpected console statement. [Error/no-console]
```

``console.log`` is an essential part of tour tests

**Comment by yelizariev** [comment]
Anyway, travis is green

File: event_sale_confirmation/models/event_registration.py (hunk 3 above)
Comment (chienandalu) [reply event_sale_confirmation/models/event_registration.py:14]: True that :+1:

**Pull request closed without merging**